stock_force_assign_by_type/readme/CONTEXT.rst

```diff
@@ -0,0 +1,7 @@
+* Forced stock reservation: Allows you to configure picking operation types so that stock
```

Comment:
This file is for putting the reasons to develop this module.

Reply:
@pilarvargas-tecnativa something along these lines: https://github.com/OCA/stock-logistics-workflow/pull/1500#discussion_r1482610850

Reply:
thanks